sql/changes/1.6/menu_item.sql

```diff
@@ -0,0 +1,11 @@
+DROP TYPE IF EXISTS menu_item CASCADE;
+CREATE TYPE menu_item AS (

```

Comment:
The same query also returns the children to prevent recomputing in the UI. We need to go through them in SQL anyway.